### ISSUE TITLE
Data Visualization Exercise

### ISSUE THREAD

**Issue by hayhem** [chr66475/connorross-MADA-portfolio#3]
Connor, nice job! I think that my best piece of advice would be to change your color palette. I have a few approaches you might take with this. One would be to change from a continuous palette to a sequential color palette (and to change the legend to ranges accordingly (0-10, 10-20, etc).`RColorBrewer` has some color palettes that I think would work nicely. Alternatively, you could keep the continuous color palette and not break the counts up into ranges, but use a continuous palette that transitions into more colors (palettes in `viridis` are great for this!)

Also, if you're interested in making maps with Hawaii and Alaska, I've worked a bit with `usmaps`. The package works on its own, but really shines when you use it with ggplot2.

**Comment by chr66475** [comment]
Sweet! Thanks Hayley! I'm a little familiar with RColorBrewer and palettes in viridis, however, I've hardly ever used them but this would be a perfect opportunity to learn. I also want to maybe switch to plotly/ggploty or leaflet to build a more interactive component in the map. I just go frustrated with it because I kept receiving error when attempting to load a map to use. I don't plan to put this down until I've figured it out :D
